src/style.css

```diff
@@ -0,0 +1,131 @@
+:root {
+  --color-text-first: #060b35;
+  --color-text-second: #616070;
+  --color-btn: #00acdc;
+  --color-main: #f3f3f3;
+}
+
+.card-page {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--color-text-first);
+}
+
+.block {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.product-card {
+  width: 198px;
```

Comment:
see, if you've had proper box-sizing (border-box), which is set under `*` selector, this value would be 200px